osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -70,6 +72,8 @@ public abstract partial class SongSelect : OsuScreen, IKeyBindingHandler<GlobalA
 
         public override bool ShowFooter => true;
 
+        public bool AllowModSelection { get; init; } = true; // TODO: use this for special song select instances like multiplayer.
```

Comment:
I think I'd drop this at least for the time being, it feels like a much-of-muchness. In my head `AllowModSelection` existing *and* `CreateFooterButtons()` being virtual is not good - you can accidentally neuter the flag by overriding the virtual. I'd need to see more than that one usage of the flag to be convinced and even then I'm not sure I could be.

I'd hope implementations can handle this just via the virtual like current song select seems to do - *although* I hope that we can avoid weird crimes such as https://github.com/ppy/osu/blob/45d248cd26ee4d9ab000249b95f4b8fd55dfd161/osu.Game/Screens/OnlinePlay/OnlinePlayFreestyleSelect.cs#L85-L86 in the new one...

Reply:
done